src/ConfigurationContext.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FunctionComponent, createContext, useContext } from "react"
+import { Platform } from "react-native"
+import env from "react-native-config"
+
+export interface Configuration {
+  displaySelfAssessment: boolean
+  displayReportAnIssue: boolean
+  appDownloadLink: string
+  authorityPrivacyPolicyUrl: string
+  authorityName: string
+  authorityAdviceUrl: string
+}
+
+const initialState = {
+  displaySelfAssessment: false,
+  displayReportAnIssue: false,
+  appDownloadLink: "",
+  authorityPrivacyPolicyUrl: "",
+  authorityName: "",
+  authorityAdviceUrl: "",
+}
+
+const ConfigurationContext = createContext<Configuration>(initialState)
+
+const DEFAULT_DOWNLOAD_LINK = ""
+const DEFAULT_AUTHORITY_PRIVACY_URL = ""
+const DEFAULT_AUTHORITY_NAME = ""
+const DEFAULT_AUTHORITY_ADVICE_URL = ""
+
+const ConfigurationProvider: FunctionComponent = ({ children }) => {
```

Comment:
Another benefit of having this here is that we could pivot, or enrich our use of the config package with external or a static file we can set on our setup scripts